spec/CloudCode.Validator.spec.js

```diff
@@ -1428,4 +1428,78 @@ describe('cloud validator', () => {
       done();
     }
   });
+
+  it('set params options function async', async () => {
+    Parse.Cloud.define(
+      'hello',
+      () => {
+        return 'Hello world!';
+      },
+      {
+        fields: {
+          data: {
+            type: String,
+            required: true,
+            options: async val => {
+              await new Promise(resolve => {
+                setTimeout(resolve, 500);
+              });
+              return val === 'f';
+            },
+            error: 'Validation failed.',
+          },
+        },
+      }
+    );
+    try {
+      await Parse.Cloud.run('hello', { data: 'd' });
+    } catch (error) {
+      expect(error.code).toEqual(Parse.Error.VALIDATION_ERROR);
```

Comment:
Change test design, because if there is no error thrown, the `expect` are never reached.